doc/build/core/constraints.rst

```diff
@@ -725,20 +733,32 @@ which works nicely with :class:`.SchemaType` since these constraints have
 only one column::
 
     metadata = MetaData(
-        naming_convention={"ck": "ck_%(table_name)s_%(column_0_name)s"}
+        naming_convention={"ck": "ck_%(table_name)s_%(constraint_name)s",
```

Comment:
I'd change the code formatting here, if you want, we can just pull this into gerrit and I can push additional tweaks and whatnot.

Reply:
let me handle most of this in github for a bit first. i still have issues with gerrit and I'll lose too much time on getting up-to-speed.